easybuild/easyblocks/s/scipy.py

```diff
@@ -198,6 +198,18 @@ def install_step(self):
         """Custom install step for scipy: use ninja for scipy >= 1.9.0"""
         if self.use_meson:
             MesonNinja.install_step(self)
+
+            # copy PKG-INFO file included in scipy source tarball to scipy-<version>.egg-info in installation,
+            # so pip is aware of the scipy installation (required for 'pip list', 'pip check', etc.);
+            # see also https://github.com/easybuilders/easybuild-easyblocks/issues/2901
+            pkg_info = 'PKG-INFO'
+            target_egg_info = os.path.join(self.installdir, self.pylibdir, 'scipy-%s.egg-info' % self.version)
+            if os.path.isfile(pkg_info):
```

Comment:
```
WARNING: PKG-INFO not found in /dev/shm/branfosj/build-up-EL8/SciPybundle/2023.02/gfbf-2022b/easybuild_obj_0, so can't use it to create
/rds/projects/2017/branfosj-rse/easybuild/EL8-ice/software/SciPy-bundle/2023.02-gfbf-2022b/lib/python3.10/site-packages/scipy-1.10.1.egg-info!
```

We are in the wrong directory.

Reply:
Oh boy, overlook that part... OK, will fix.

Reply:
```py
pkg_info = os.path.join(self.builddir, self.name, '%s-%s' % (self.name, self.version), 'PKG-INFO')
```

I am not sure if there is a better option than that for getting to that file.

Reply:
can we use `self.cfg['start_dir']` or something like that?

Reply:
Yes. That was what I missed earlier:

```py
pkg_info = os.path.join(self.cfg['start_dir'], 'PKG-INFO')
```

Reply:
yeah just tested it and that works

Reply:
done in bd9e693